.storybook/preview.tsx

```diff
@@ -29,8 +29,8 @@ export const decorators = [
   Story => (
     <SiteProvider>
       <LocaleProvider>
-        <ThemeProvider font={openSans.style.fontFamily}>
-          <div data-test-id="story-root">
+        <ThemeProvider>
+          <div data-test-id="story-root" className={openSans.className}>
```

Comment:
```suggestion
          <div data-test-id="story-root" className={openSans.variable}>
```

Reply:
Keep in mind that once we move to `pages/_app.tsx` we should pretty much define the font variable there.